app-emulation/fuse/metadata.xml

```diff
@@ -12,4 +12,21 @@
 	<upstream>
 		<remote-id type="sourceforge">fuse-emulator</remote-id>
 	</upstream>
+	<use>
+		<flag name="backend-fbcon">
```

Comment:
Could you please squash this a bit and make each `<flag>...</flag>` take a single line? The amount of vertical space used here is unnecessary given the length of messages conveyed.